.kokoro

```diff
@@ -34,10 +34,14 @@ popd
 
 if [ -n "$KOKORO_BUILD_NUMBER" ]; then
   bazel version
-  use_bazel.sh 0.20.0
+  if [ -f "$KOKORO_GFILE_DIR/use_bazel.sh" ]; then
+    bash "$KOKORO_GFILE_DIR/use_bazel.sh" 1.1.0
+  else
+    use_bazel.sh 1.1.0
+  fi
   bazel version
 fi
 
-./.kokoro-ios-runner/bazel.sh test //:UnitTests 8.1.0
+./.kokoro-ios-runner/bazel.sh test //:UnitTests 9.0.0
```

Comment:
Is `9.0.0` a version number? Would you please put it in a variable and document the variable?

Reply:
Done.